app/cdap/text/text-en.yaml

```diff
@@ -3185,6 +3185,7 @@ features:
       searchLabel: Search by batch pipeline name
       tab: Local pipelines
     table:
+      connected: Connected to Git
       pipelineName: Pipeline name
       lastSyncTime: Last sync time
       gitStatus: Git status
```

Comment:
Guess we should remove `lastSyncTime` and `filePathInGitRepo` since they're not used anymore?
Also let's re-word `Git status` to something like `pushed to Git`? perhaps need product info